ui/static/ui/js/learning_resources.jsx

```diff
@@ -6,73 +6,179 @@ define('learning_resources', [
   var StatusBox = Utils.StatusBox;
   var Select2 = Utils.Select2;
 
-  var VocabularyOption = React.createClass({
+  var TermList = React.createClass({
     render: function () {
-      var options = _.map(this.props.terms, function (term) {
+      var appliedVocabularies = this.props.vocabs.map(function (vocab) {
+        var selectedTerms = vocab.selectedTerms;
+        var vocabularyName = vocab.vocabulary.name;
+
+        if (selectedTerms.length) {
+          return (
+            <TermListItem
+              key={vocab.vocabulary.id}
+              label={vocabularyName}
+              terms={selectedTerms.join(", ")}
+            />
+          );
+        }
+      });
+      return (
+        <div id="term-list-container" className="panel panel-default">
+          <div className="panel-heading">
+              Taxonomy Terms applied to this Learning Resource
+          </div>
+          <ul id="term-list" className="list-group">
+            {appliedVocabularies}
+          </ul>
+        </div>
+      );
+    }
+  });
+
+  var TermListItem = React.createClass({
+    render: function () {
+      return (
+        <li className="applied-term list-group-item">
+          <strong>{this.props.label}: </strong> {this.props.terms}
+        </li>
+      );
+    }
+  });
+
+  var VocabSelect = React.createClass({
+    render: function () {
+      var options = [];
```

Comment:
Would it be possible to move the function argument to the next line instead? Or just use `var options;`